optimization/ParetoOptimization/matRad_paretoOptimization.m

```diff
@@ -0,0 +1,577 @@
+function returnStruct = matRad_paretoOptimization(dij,cst,pln,nIter,wInit)
```

Comment:
Why can this not be resultGUI as a return value? 
I understand that you don't get a single plan, but it could just be one of the Pareto-Optimal plans (e.g. the one having the originally set weights equivalent or the closest point to it).

Reply:
So [resultGUI, returnStruct]?